platforms/core-configuration/kotlin-dsl/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
     id("gradlebuild.kotlin-dsl-dependencies-embedded")
     id("gradlebuild.kotlin-dsl-sam-with-receiver")
     id("gradlebuild.kotlin-dsl-plugin-bundle-integ-tests")
-    id("com.gradleup.shadow").version("9.0.0-beta11")
+    id("com.gradleup.shadow").version("9.1.0")
```

Comment:
I'm afraid you forgot to set `duplicatesStrategy` for `tasks.shadowJar`, see https://gradleup.com/shadow/configuration/merging/#handling-duplicates-strategy